.github/workflows/setup-gcloud-it.yml

```diff
@@ -396,8 +394,6 @@ jobs:
         uses: ./
         with:
           version: 'latest'
-          service_account_email: ${{ secrets.SETUP_GCLOUD_IT_EMAIL }}
```

Comment:
Why remove that?

Reply:
it wasnt being used in the code anywhere